kanidmd/Cargo.toml

```diff
@@ -51,6 +51,7 @@ serde_cbor = "0.11"
 serde_json = "1.0"
 serde_derive = "1.0"
 
+libsqlite3-sys = { version = "0.17", features = ["bundled"] }
```

Comment:
I think you forgot to update this one too? Otherwise it looks great!

Reply:
Is Kanidmd going to be distributed as package or only as a container? I think that containers are the standard for this kind of services and if is only a container we don't need to support both compilations, but as you prefer.

Reply:
It may be packaged, and already a package exists for suse, but I haven't really advertised it much. So I think we should be consistent in our configuration here.

Reply:
Greats. Here is it.